src/Raven.Server/Web/Studio/UpgradeInfoHandler.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Raven.Server.Commercial;
+using Raven.Server.Json;
+using Raven.Server.Routing;
+using Raven.Server.ServerWide;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Web.Studio;
+
+public sealed class UpgradeInfoHandler : ServerRequestHandler
+{
+    [RavenAction("/studio/upgrade-info", "GET", AuthorizationStatus.ValidUser, EndpointType.Read)]
+    public async Task GetUpgradeInfo()
+    {
+        if (ServerVersion.Build == ServerVersion.DevBuildNumber)
+        {
+            HttpContext.Response.StatusCode = (int)HttpStatusCode.NoContent;
+            return;
+        }
+            
+        var licenseId = ServerStore.LoadLicense()?.Id.ToString();
```

Comment:
Do we need to load the license? The license manager should already have its ID: `ServerStore.LicenseManager.LicenseStatus.Id`

Reply:
Fixed